### ISSUE TITLE
Use coverage to detect test-production classes dependencies

### ISSUE THREAD

**Issue by lordofthejars** [arquillian/smart-testing#215]
##### Issue Overview

Currently affected strategy just works with black box testing(or white ones if ShrinkWrap or VertX runner is used). In #214 we mention to  create a file where developer can set the relationship between test and production code under test. We could automate this by using coverage information.

**Comment by MatousJobanek** [comment]
> Currently affected strategy just works with black box testing(or white ones if ShrinkWrap or VertX runner is used). 

it is the other way around - currently, it works with white box tests, not black box, right?

**Comment by lordofthejars** [comment]
Yes it does not work with black box if vertx or shrinkwrap not used

**Comment by lordofthejars** [comment]
Related to https://github.com/arquillian/smart-testing/issues/264
